src/java/picard/sam/ReplaceSamHeader.java

```diff
@@ -48,14 +48,26 @@
  * @author alecw@broadinstitute.org
  */
 @CommandLineProgramProperties(
-        usage = "Replace the SAMFileHeader in a SAM file with the given header. " +
-                "Validation is minimal.  It is up to the user to ensure that all the elements referred to in the SAMRecords " +
-                "are present in the new header.  Sort order of the two input files must be the same.",
-        usageShort = "Replace the SAMFileHeader in a SAM file with the given header",
+        usage =  ReplaceSamHeader.USAGE_SUMMARY + ReplaceSamHeader.USAGE_DETAILS,
+        usageShort = ReplaceSamHeader.USAGE_SUMMARY,
         programGroup = SamOrBam.class
 )
 public class ReplaceSamHeader extends CommandLineProgram {
-
+    static final String USAGE_SUMMARY = "Replace the SAMFileHeader in a \".sam\" or \".bam\" file.  ";
+    static final String USAGE_DETAILS = "Validation is minimal.  " +
```

Comment:
"Replace" -> "Replaces"

"Validation is minimal. It is up to the user" -> "Note that header validation is minimal, so it is up to the user"